modular_skyrat/modules/assault_operatives/code/turrets.dm

```diff
@@ -0,0 +1,32 @@
+//TURRETS>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+/obj/machinery/porta_turret/assaultops
+	use_power = IDLE_POWER_USE
+	req_access = list(ACCESS_SYNDICATE)
+	faction = list(ROLE_SYNDICATE)
+	mode = TURRET_STUN
+	max_integrity = 200
+	base_icon_state = "syndie"
+	stun_projectile = /obj/projectile/energy/electrode
+	stun_projectile_sound = 'sound/weapons/taser.ogg'
+	lethal_projectile = /obj/projectile/beam/laser/heavylaser
+	lethal_projectile_sound = 'sound/weapons/lasercannonfire.ogg'
+
+/obj/machinery/porta_turret/assaultops/assess_perp(mob/living/carbon/human/perp)
+	return 10
```

Comment:
That's probably supposed to be a define here.

Reply:
Nope! It's a broken ass sytem!s